analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DatabasePasswordsShouldBeSecure.NetCore31.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace TestFrameworkExtensions
+{
+    using Microsoft.EntityFrameworkCore;
+
+    public class NoncompliantDbContext : DbContext
+    {
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer(); // Error [CS1501]
+            optionsBuilder.UseNpgsql(); // Error [CS1501]
+            optionsBuilder.UseSqlite(); // Error [CS1501]
+            optionsBuilder.UseOracle(); // Error [CS1501]
+            optionsBuilder.UseMySQL(); // Error [CS1501]
```

Comment:
This cases are added just to increase coverage?

Reply:
Yes. And I added one for each DB provider to document the existing difference between netcore31 and net5 between APIs.